### ISSUE TITLE
Running AppImage on Fedora 41 in Gnome causes screen reader to enable and then fails to load

### ISSUE THREAD

**Issue by lvturner** [SoftFever/OrcaSlicer#6857]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### OrcaSlicer Version

2.1.0

### Operating System (OS)

Linux

### OS Version

Fedora 41

### Additional system information

OS: Fedora Linux 41 (Workstation Editi 
Host: N7 B650E 
Kernel: 6.11.0-63.fc41.x86_64 
Uptime: 2 hours, 10 mins 
Packages: 2626 (rpm), 56 (flatpak) 
Shell: fish 3.7.0 
Resolution: 3840x2160 
 DE: GNOME 47.0 
WM: Mutter 
WM Theme: Adwaita 
Theme: Adwaita [GTK2/3] 
Icons: Adwaita [GTK2/3] 
Terminal: ptyxis-agent 
CPU: AMD Ryzen 7 7800X3D (16) @ 5.050G 
GPU: NVIDIA GeForce RTX 4090 
GPU: AMD ATI 79:00.0 Raphael 
Memory: 7256MiB / 63369MiB 

### Printer

N/A

### How to reproduce

I downloaded and ran OrcaSlicer_Linux_V2.1.0 -- I used AppImageLauncher (https://www.gnome-look.org/p/1228228/) to install and run it and it failed to load.

Trying to run from terminal and Gnome screen reader was suddenly enabled.

There was the following output on the terminal

Warning:          Could not load keyboard geometry for :0
                  BadName (named color or font does not exist)
                  Resulting keymap file will not describe geometry

### Actual results

Screen reader enables and orca slicers doesn't load

### Expected results

Orca slicer running successfully

### Project file & Debug log uploads

N/A

### Checklist of files to include

- [ ] Log file
- [ ] Project file

**Comment by CitrusIntellect** [comment]
I dont know how buggy the app image is but I know it is buggy. If you can try compiling a git clone of the Repo. If you have to run with an AppImage, try these commands

chmod +x [filename].AppImage
./[filename].AppImage
